modules/javafx.graphics/src/test/addExports

```diff
@@ -56,6 +56,7 @@
 --add-opens javafx.graphics/javafx.scene.layout=ALL-UNNAMED
 --add-opens javafx.graphics/javafx.scene.paint=ALL-UNNAMED
 --add-opens javafx.graphics/javafx.stage=ALL-UNNAMED
+--add-opens java.desktop/javax.imageio=ALL-UNNAMED
```

Comment:
this change seems unrelated

Reply:
`WindowControlsOverlayTest.activePseudoClassCorrespondsToStageFocusedProperty()` requires `opens javafx.stage`.

Reply:
the question is about `java.desktop/javax.imageio` not `javafx.graphics/javafx.stage` or am I missing something?

Reply:
Not sure why GitHub highlights this line as changed, because it's not. The line that has actually changed is
`--add-opens javafx.graphics/javafx.stage=ALL-UNNAMED`.